src/toast/toast.container.js

```diff
@@ -0,0 +1,22 @@
+import { connect } from 'react-redux'
+import Component from './toast'
+
+import { getCurrentToast, close, remove } from './redux'
+
+const mapStateToProps = (state, { position }) => {
```

Comment:
You don't have to get a props from `ownProps`to inject it again to `stateProps`.
At the end `connect` from *react-redux* will do this: 
```es6
mergeProps = (stateProps, dispatchProps, ownProps) => ({
  ...stateProps,
  ...dispatchProps,
  ...ownProps,
})
```

Reply:
👍